sub-logs/wbp.json

```diff
@@ -0,0 +1,15 @@
+{
+    "owner": {
+        "username": "whizbangpop",
+        "email": "abi@quickexit.cc"
+    },
+
+    "target": {
+        "CNAME": {
+            "name": "wbp",
+            "value": "cname.short.io"
```

Comment:
I'm getting the following error when trying to visit the CNAME: `NET::ERR_CERT_AUTHORITY_INVALID`

Could you look into that?

Reply:
@DEV-DIBSTER It is a URL shortener and does not have a website or an SSL for that domain.